release-notes.md

```diff
@@ -38,37 +38,45 @@ The following updates were made in this release:
 * Added enhanced alert reporting with clickthrough URL support in the Campaign response event.
 * Added support for a custom title in local notifications.
 
+
+
+## May 30, 2019
+
+**Android Places Monitor 1.0.1**
+
+- Fixed bug that prevents entry event for pois happening on launch.
```

Comment:
Please reword so that it reads as follows:

"Fixed an issue that prevented an entry event for POIs when the Android Monitor extension is started."